org.eclipse.winery.repository.ui/src/app/instance/artifactTemplates/artifactSource/artifactSource.component.ts

```diff
@@ -0,0 +1,231 @@
+/**
+ * Copyright (c) 2017 University of Stuttgart.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * and the Apache License 2.0 which both accompany this distribution,
+ * and are available at http://www.eclipse.org/legal/epl-v20.html
+ * and http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Contributors:
+ *     Philipp Meyer - initial API and implementation
```

Comment:
This is a new class. Please do not list contributors. They are not required at EPLv2 any more. See https://dev.eclipse.org/mhonarc/lists/winery-dev/msg00053.html